src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -280,23 +287,26 @@ private boolean isLetterHolderRole(String role) {
         }
 
         User letterHolder = userService.getUser(authorisation);
-        return Optional.of(readCase(letterHolder, letterHolderCases.get(0)));
+        CaseDetails caseDetails = readCase(letterHolder, letterHolderCases.get(0));
+        if (CREATE.getValue().equals(caseDetails.getState())) {
```

Comment:
It's enum description value, should be same all the time, no need for equalsIgnoreCase

Reply:
`caseDetails.getState()` -- is a String

Reply:
`caseDetails.getState()` is coming from the CCD definition and we should not change again. In my opinion, if that is messed up I would rather like it to break our functional tests rather than ignoring case and let them pass.

Reply:
If we are confident, its always convert the state to Enum and compare with Enum. We created Enum and comparing as String seems wrong.

Reply:
This what the story says **Defendant dashboards don't show cases that haven't been OPEN yet.**
I am not sure how the code and the story are both correct

Reply:
> `caseDetails.getState()` is coming from the CCD definition and we should not change again. In my opinion, if that is messed up I would rather like it to break our functional tests rather than ignoring case and let them pass.

I am not saying we should change it .. I am just saying the it doesn't have to be casesensitive..

Reply:
anyways this is just a suggestion.. you can skip this if you don't want to change it.